containers/Loans/getOpenLoans.tsx

```diff
@@ -0,0 +1,90 @@
+import { BigNumber, ethers, providers } from 'ethers';
+import { Network } from 'store/wallet';
+import { getInfuraRpcURL } from 'utils/infura';
+import getLoan from './getLoan';
+import { Loan } from './types';
+
+/**
+ * Note, once the subgraphs has synced up on optimism it would make sense to use those instead the events log
```

Comment:
isn't the graph synced on L2?

Reply:
Yea there were some issue and noah said it would take a few weeks to sycn

Reply:
yes but I heard that only affects the graph on L1 but that could be already old information

Reply:
I think I discovered the issue, this query should return closed and open loans:
```
{
  loans(first: 5) {
    id
    isOpen
  }
}
```
And I get an empty array back https://thegraph.com/hosted-service/subgraph/synthetixio-team/optimism-main?selected=playground